FRC1076-2016-Competition/test/org/usfirst/frc/team1076/test/sensor/DistanceEncoderTest.java

```diff
@@ -0,0 +1,93 @@
+package org.usfirst.frc.team1076.test.sensor;
+
+import static org.junit.Assert.*;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.usfirst.frc.team1076.robot.sensors.DistanceEncoder;
+import org.usfirst.frc.team1076.robot.sensors.GearShiftStateManager.GearStates;
+import org.usfirst.frc.team1076.test.mock.MockEncoder;
+import org.usfirst.frc.team1076.test.mock.MockGearShifter;
+import org.usfirst.frc.team1076.test.mock.MockRobot;
+
+public class DistanceEncoderTest {
+	
+	private static final double EPSILON = 1e-5;
+	// Constant to make the assertions look nicer.
+	private static final double SCALING = 4096 * 6 * Math.PI;
```

Comment:
Change `SCALING` to be a constant queriable from the `DistanceEncoder`, perhaps including the gear ratio. Add tests for the gear ratio numbers.